examples/i256/src/main.rs

```diff
@@ -14,10 +15,10 @@ pub fn get_matrix(val: i32) -> Matrix {
 
 pub fn mult(a: &Matrix, b: &Matrix) -> Matrix {
     let mut c = get_matrix(0);
-    for i in 0..N {
-        for j in 0..N {
-            for k in 0..N {
-                c[i][j] += &a[i][k] * &b[k][j];
+    for (i, row_a) in a.iter().enumerate() {
```

Comment:
I know clippy complains but in this case could we put an `allow` just for readability?

Reply:
yup